src/xunit.netcore.extensions/Discoverers/ActiveIssueDiscoverer.cs

```diff
@@ -28,17 +28,40 @@ public class ActiveIssueDiscoverer : ITraitDiscoverer
             Debug.Assert(ctorArgs.Count() >= 2);
 
             string issue = ctorArgs.First().ToString();
-            TestPlatforms platforms = (TestPlatforms)ctorArgs.Last();
+            TestPlatforms platforms = TestPlatforms.Any;
+            TargetFrameworkMonikers frameworks = (TargetFrameworkMonikers)~0;
+            
+            foreach (object arg in ctorArgs.Skip(1)) // First argument is the issue number.
+            {
+                if (arg is TestPlatforms)
+                {
+                    platforms = (TestPlatforms)arg;
+                }
+                else
+                {
+                    frameworks = (TargetFrameworkMonikers)arg;
+                }
+            }
+        
             if ((platforms.HasFlag(TestPlatforms.FreeBSD) && RuntimeInformation.IsOSPlatform(OSPlatform.Create("FREEBSD"))) ||
                 (platforms.HasFlag(TestPlatforms.Linux) && RuntimeInformation.IsOSPlatform(OSPlatform.Linux)) ||
                 (platforms.HasFlag(TestPlatforms.NetBSD) && RuntimeInformation.IsOSPlatform(OSPlatform.Create("NETBSD"))) ||
                 (platforms.HasFlag(TestPlatforms.OSX) && RuntimeInformation.IsOSPlatform(OSPlatform.OSX)) ||
                 (platforms.HasFlag(TestPlatforms.Windows) && RuntimeInformation.IsOSPlatform(OSPlatform.Windows)))
             {
-                yield return new KeyValuePair<string, string>(XunitConstants.Category, XunitConstants.Failing);
```

Comment:
I'm using ~0 as a default so that it has all of the frameworks' flag, so when doing if(framework.HasFlag) for all of the TargetFrameworkMonikers is going to be true and will return to skip the tests for that platform. Either way works fine, I've tested it with both. Which one you prefer? With 0 as might be more readable?